### PR TITLE
Batch payments

### PR DESCRIPTION
**Pull request by wnowicki**
### Added
- Batch Payments (alpha)

### CODE REVIEW

File: _includes/v4/installations.md [hunk 1]
```diff
@@ -274,3 +274,147 @@ Name | Required | Type | Description
...
+`$.id`           | Yes      | int    | Batch ID
+`$.status`       | Yes      | string | Batch status
+`$.payments.[]`  | Yes      | list   |
+`$.requested_at` | Yes      | string (ISO8601 Formatted date) |
+
+##### Payment
+
+Name               | Required | Type   | Description
+-------------------|----------|--------|-------------
+`$.application`    | Yes      | int    | Batch ID
```
Comment (sped2773) [inline _includes/v4/installations.md]: typos - should be application id and payment status

File: _includes/v4/installations.md [hunk 2]
```diff
@@ -274,3 +274,147 @@ Name | Required | Type | Description
...
+It is possible to add merchant payments to applications, by making a call to the merchant payment service. You may add any amount of payment that is `>0`, as the system does not consider a payment of 0p to be a payment.
+
+#### Parameters
+
+Name | Required | Type | Description
+--- | --- | --- | ---
+`$.payments`  | Yes   | list | List of payments to be added
+`$.payments.[].application`  | Yes  | int |  
+`$.payments.[].amount` | Yes | int | Payment amount in *pence*  and must be `>0`
+`$.payments.[].effective_date` | Yes | string (ISO8601 Formatted date) | e.g '2016-12-31'
```
Comment (KrekoG) [inline _includes/v4/installations.md]: For your consideration: we are linking to the wiki-page on many parts of the api reference page when we are specifying ISO 8601.
`[ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)`

File: _includes/v4/installations.md [hunk 3]
```diff
@@ -274,3 +274,147 @@ Name | Required | Type | Description
     }
 ]
 ```
+
+## Batch Payments
+
+> **Alpha Feature**  
+> Please note that this feature is Alpha state is available for a limited partners and it's still under development so may change
```
Comment (EvanBarbour3) [inline _includes/v4/installations.md]: > Please note that this feature is in Alpha state, and only available for a limited number of partners. It is considered under development and may be subject to change

File: _includes/v4/installations.md [hunk 4]
```diff
@@ -274,3 +274,147 @@ Name | Required | Type | Description
...
+
+### Add Batch
+
+```
+POST {{ site.data.globals.api_prefix }}/installations/:installation/merchant-payments
+```
+
+It is possible to add merchant payments to applications, by making a call to the merchant payment service. You may add any amount of payment that is `>0`, as the system does not consider a payment of 0p to be a payment.
```
Comment (EvanBarbour3) [inline _includes/v4/installations.md]: `by making an API call`

`as the system does not consider anything less to be a payment`

File: _includes/v4/installations.md [hunk 5]
```diff
@@ -274,3 +274,147 @@ Name | Required | Type | Description
...
+
+#### Parameters
+
+Name | Required | Type | Description
+--- | --- | --- | ---
+`$.payments`  | Yes   | list | List of payments to be added
+`$.payments.[].application`  | Yes  | int |  
+`$.payments.[].amount` | Yes | int | Payment amount in *pence*  and must be `>0`
```
Comment (EvanBarbour3) [inline _includes/v4/installations.md]: Payment amount in *pence*, must be `>0`

(you had a double space between 'pence' & 'and'

**Pull request merged**